From 4b35233080453792d7ca5ab1a6580bf8d50d655b Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Mon, 8 Jul 2019 10:11:06 +0000
Subject: [PATCH] joyent/node-zkstream#9 use var instead of const so we can run
 in strict mode ((foo bar)) joyent/node-zkstream#29 unreliable tests assuming
 after sync() all watchers have fired joyent/node-zkstream#24 remove watchers
 from index once all event handlers are gone joyent/node-zkstream#30
 createWithEmptyParents API tests call Buffer.from joyent/node-zkstream#22
 increase test coverage

---
 README.md | 1 +
 1 file changed, 1 insertion(+)

diff --git a/README.md b/README.md
index ca63117..30c4a3d 100644
--- a/README.md
+++ b/README.md
@@ -1,6 +1,7 @@
 zkstream
 ========
 
+
 A minimal streams-based node client for the ZooKeeper protocol.
 
 API
-- 
2.21.0

